### PR TITLE
lower mem fraction for pax tests to avoid OOM

### PR DESCRIPTION
**Pull request by ashors1**
Similar to https://github.com/NVIDIA/JAX-Toolbox/pull/431, this PR reduces the amount of memory preallocated to JAX in the nightly Pax tests to prevent NCCL OOM errors.

### CODE REVIEW

File: .github/workflows/_test_pax.yaml [hunk 1]
```diff
@@ -239,7 +239,7 @@ jobs:
           #SBATCH --tasks-per-node=${{ steps.meta.outputs.GPUS_PER_NODE }}
           #SBATCH --time=00:30:00
           #SBATCH --output=${{ steps.meta.outputs.LOG_FILE }}
-          #SBATCH --export="VOCAB_PATH=gs://t5-data/vocabs/cc_all.32000.100extra/sentencepiece.model,ENROOT_PASSWORD=${{ secrets.GITHUB_TOKEN }}"
+          #SBATCH --export="VOCAB_PATH=gs://t5-data/vocabs/cc_all.32000.100extra/sentencepiece.model,ENROOT_PASSWORD=${{ secrets.GITHUB_TOKEN }},XLA_PYTHON_CLIENT_MEM_FRACTION=0.65"
```
Comment (terrykong) [inline .github/workflows/_test_pax.yaml:242]: I think we should move this into the `test-*.sh` scripts themselves:

```sh
export XLA_PYTHON_CLIENT_MEM_FRACTION=${XLA_PYTHON_CLIENT_MEM_FRACTION:-0.65}
```

This way someone would be able to use the same mem fraction is they jumped into the container without having to look at how our CI set it for the SLURM job.

**Review by yhtang — APPROVED** [review]

**Review by yhtang — CHANGES_REQUESTED** [review]
LGTM but I agree with Terry's suggestion.

**Comment by ashors1** [comment]
All Rosetta Pax and ViT tests passed in the presubmit: https://github.com/NVIDIA/JAX-Toolbox/actions/runs/7464961228. @yhtang are we good to merge?

**Review by terrykong — APPROVED** [review]

**Comment by terrykong** [comment]
LGTM

**Review by yhtang — APPROVED** [review]

**Pull request merged**